app/assets/javascripts/discourse/app/components/user-menu/menu.hbs

```diff
@@ -30,6 +30,12 @@
             />
           {{/each}}
         </div>
+
+        <PluginOutlet
+          @name="user-menu-tabs-after"
```

Comment:
Most examples I see in core for before/after outlets have the placement identifier at the front of the name. IE.

` @name="after-create-topic-button"`
`@name="above-latest-topic-list-item"`
`@name="below-latest-topic-list-item-bottom-row"`

Reply:
> So the name should probably be after-user-menu-tabs

I'd suggest calling this plugin outlet `user-menu-tabs-list__after`.

`user-menu-tabs-list` identifies the placement and using the suffix`__after` ensures it's compatible with a wrapper plugin outlet, should we choose to insert one here in the future.

Reply:
done, thanks